source/administration/concepts.rst

```diff
@@ -0,0 +1,162 @@
+============================
+Core Administration Concepts
+============================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+The following core concepts are fundamental to the administration of MinIO deployments, including but not limited to object retention, encryption, and access management.
+
+What Is Object Storage?
+-----------------------
+
+.. _objects:
+
+An :ref:`object <objects>` is binary data, sometimes referred to as a Binary Large OBject (BLOB). 
+Blobs can be images, audio files, spreadsheets, or even binary executable code. 
+Object Storage platforms like MinIO provide dedicated tools and capabilities for storing, retrieving, and searching for blobs. 
+
+.. _buckets:
+
+MinIO Object Storage uses :ref:`buckets <buckets>` to organize objects. 
+A bucket is similar to a folder or directory in a filesystem, where each bucket can hold an arbitrary number of objects. 
+MinIO buckets provide the same functionality as AWS S3 buckets. 
+
+For example, consider an application that hosts a web blog. 
+The application needs to store a variety of blobs, including rich multimedia like videos and images. 
+The structure of objects on the MinIO server might look similar to the following:
+
+.. code-block:: text
+
+   / #root
+   /images/
+      2022-01-02-MinIO-Diagram.png
+      2022-01-03-MinIO-Advanced-Deployment.png
+      MinIO-Logo.png
+   /videos/
+      2022-01-04-MinIO-Interview.mp4
+   /articles/
+      /john.doe/
+         2022-01-02-MinIO-Object-Storage.md
+         2022-01-02-MinIO-Object-Storage-comments.json
+      /jane.doe/
+         2022-01-03-MinIO-Advanced-Deployment.png
+         2022-01-02-MinIO-Advanced-Deployment-comments.json
+         2022-01-04-MinIO-Interview.md
```

Comment:
Suggest dropping these examples and using them when we talk about prefixing, as it makes way more sense there.

Reply:
I like this idea. I've replaced the example in the prefix section with this content.